src/components/common/Dialogs/AwardAndSmiteDialogs/ManageReputationContainer/ManageReputationContainer.tsx

```diff
@@ -0,0 +1,186 @@
+import React, { useState } from 'react';
+import { string, object, number, boolean, InferType } from 'yup';
+import { Id } from '@colony/colony-js';
+import { useNavigate } from 'react-router-dom';
+import { defineMessages } from 'react-intl';
+
+import Decimal from 'decimal.js';
+import Dialog from '~shared/Dialog';
+import { ActionHookForm as Form } from '~shared/Fields';
+
+import { DEFAULT_TOKEN_DECIMALS } from '~constants';
+import { ActionTypes } from '~redux/index';
+import { pipe, withMeta, mapPayload } from '~utils/actions';
+import { useEnabledExtensions } from '~hooks'; // useSelectedUser
+// import { getVerifiedUsers } from '~utils/verifiedRecipients';
+import { notNull } from '~utils/arrays';
+
+import DialogForm from '../ManageReputationDialogForm';
+import { AwardAndSmiteDialogProps } from '../types';
+
+import { getManageReputationDialogPayload } from './helpers';
+
+const displayName = 'common.ManageReputationContainer';
+
+const MSG = defineMessages({
+  amountZero: {
+    id: `${displayName}.amountZero`,
+    defaultMessage: 'Amount must be greater than zero',
+  },
+  maxAmount: {
+    id: `${displayName}.maxAmount`,
+    defaultMessage: "Amount must be less than the user's reputation",
+  },
+});
+
+const defaultValidationSchema = object()
+  .shape({
+    domainId: number().required(),
+    user: object().shape({
+      walletAddress: string().address().required(),
+    }),
+    amount: string()
+      .required()
+      .test(
+        'more-than-zero',
+        () => MSG.amountZero,
+        (value) => {
+          const numberWithoutCommas = (value || '0').replace(/,/g, '');
+          return !new Decimal(numberWithoutCommas).isZero();
+        },
+      ),
+    annotation: string().max(4000),
+    forceAction: boolean(),
+    motionDomainId: number(),
+  })
+  .defined();
+
+type FormValues = InferType<typeof defaultValidationSchema>;
+
+const ManageReputationContainer = ({
+  colony: { nativeToken, watchers },
+  colony,
+  callStep,
+  prevStep,
+  cancel,
+  close,
+  filteredDomainId,
+  isSmiteAction = false,
+}: AwardAndSmiteDialogProps) => {
+  const [isForce, setIsForce] = useState(false);
+  const [userReputation, setUserReputation] = useState(0);
+  const navigate = useNavigate();
+
+  const colonyWatchers =
+    watchers?.items.filter(notNull).map((item) => item.user) || [];
+
+  // const verifiedUsers = useMemo(() => {
+  //   return getVerifiedUsers(colony.whitelistedAddresses, colonyWatchers) || [];
+  // }, [colonyWatchers, colony]);
+
+  const updateReputationCallback = (userRepPercentage: number) => {
+    setUserReputation(userRepPercentage);
+  };
+
+  const { isVotingReputationEnabled } = useEnabledExtensions(colony);
```

Comment:
I thought you'd pass this as a prop instead of making `useEnabledExtensions` accept `colony` as its argument

Reply:
Yeah, this branch doesn't have those changes. Until now that I'm rebasing it